src/main/java/network/brightspots/rcv/GuiConfigController.java

```diff
@@ -245,18 +245,21 @@
   private Tab tabOutput;
 
   private static Provider getProviderChoice(ChoiceBox<Provider> choiceBox) {
-    return choiceBox.getValue() != null ? Provider.getByLabel(choiceBox.getValue().toString())
+    return choiceBox.getValue() != null ? Provider
+        .getByInternalLabel(choiceBox.getValue().getInternalLabel())
         : Provider.PROVIDER_UNKNOWN;
   }
 
   private static WinnerElectionMode getWinnerElectionModeChoice(
       ChoiceBox<WinnerElectionMode> choiceBox) {
     return choiceBox.getValue() != null ? WinnerElectionMode
-        .getByLabel(choiceBox.getValue().toString()) : WinnerElectionMode.MODE_UNKNOWN;
+        .getByInternalLabel(choiceBox.getValue().getInternalLabel())
+        : WinnerElectionMode.MODE_UNKNOWN;
```

Comment:
Because this calls `.toString()` it'd make it "Mode unknown" in the config file instead of "modeUnknown". Related to my other comment.

Reply:
Reading this comment again, I realize I don't understand it. This method returns a `WinnerElectionMode`, and then in `createRawContestConfig`, we call `.getInternalLabel()` on that object. So what do you mean when you say that it calls `.toString()`?

Reply:
Good point, thought it was returning a String.